.github/ISSUE_TEMPLATE/localize_widget.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Localize Widget
+about: Provide description and implementation details
+title: ''
+labels: 'localization'
+assignees: ''
+
+---
+
+**Widget ID**
+Widget ID
+
+**Widget Fields**
+Fields in Widget to localize (For example - accident_severity)
+
+**Dictionaries Missing**
+Dictionaries to create (For example - Accident Severity Dictionary)
+Create a dictionary containing using the DB code as a key and not the Hebrew translation (for example {1: 'fatal'} rather than {'קטלנית': 'fatal'})
```

Comment:
Nice! You can wrap the {1: 'fatal'} with \`  to get `{1: 'fatal'}` since code is pasted in issues very often, it will "expose" this capability to issue creator in case he's not familiar with it